mlrun/platforms/iguazio.py

```diff
@@ -367,6 +367,7 @@ def __init__(
         endpoint=None,
         access_key=None,
         mock=False,
+        **kwargs,
```

Comment:
I don't see it being used, is it kind of "hahana lemzagan" or just left over?

Reply:
It's needed to avoid a failure on extra parameters (because the same parameters are passed here and to `StreamTarget`).